packages/studio/tailwind.config.ts

```diff
@@ -1,9 +1,60 @@
 import type { Config } from "tailwindcss";
+import {
+  StudioTailwindConfig,
+  StudioTailwindTheme,
+  STUDIO_PROCESS_ARGS_OBJ,
+  CliArgs,
+} from "@yext/studio-plugin";
 import path from "path";
+import fs from "fs";
+
+const getRootDir = (): string => {
+  const cliArgs: CliArgs = JSON.parse(
+    process.env[STUDIO_PROCESS_ARGS_OBJ] as string
+  );
+  return cliArgs.root ?? process.cwd();
+};
+const rootDir = getRootDir();
+
+/**
+ * The user's StudioTailwindTheme.
+ */
+const userTailwindTheme: StudioTailwindTheme =
+  (() => {
+    try {
+      const tailwindConfigPath = path.resolve(rootDir, "tailwind.config.ts");
+      if (fs.existsSync(tailwindConfigPath)) {
+        // We have to use require() instead of a dynamic import because
+        // tailwind does not support async config.
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        const tailwindConfig: StudioTailwindConfig = require(tailwindConfigPath);
+        return tailwindConfig?.theme?.extend;
```

Comment:
For now, do we only support extensions to the Tailwind config and not customizations of existing classes?

Reply:
yup